libs/go/zmscli/cli.go

```diff
@@ -1077,6 +1077,22 @@ func (cli *Zms) EvalCommand(params []string) (*string, error) {
 			} else if argc == 2 {
 				return cli.DeleteDomainTags(dn, args[0], args[1])
 			}
+		case "put-domain-dependency":
+			if argc == 1 {
+				return cli.PutDomainDependency(dn, args[0])
+			}
+		case "delete-domain-dependency":
+			if argc == 1 {
+				return cli.DeleteDomainDependency(dn, args[0])
+			}
+		case "get-dependent-service-list":
+			if argc == 1 {
+				return cli.GetDependentServiceList(dn)
+			}
+		case "get-dependent-domain-list":
```

Comment:
because this command does not require a domain, it's in a  wrong switch statement. it should be handled in the previous switch block where commands without the domain arguments are handled.